sendr/views.py

```diff
@@ -13,24 +14,29 @@ def _get_recipients():
 
 
 class SendMail(APIView):
-    parser_classes = (parsers.MultiPartParser,)
+    parser_classes = (parsers.MultiPartParser, parsers.FormParser, )
 
     def post(self, request, format=None):
-        file_contents = self.request.FILES['image']
+        file_contents = self.request.data['image']
```

Comment:
Might want to think about making this optional?

At least having an image attachment might be optional.